solidity/contracts/bridge/library/FraudVerification.sol

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity 0.8.4;
+
+import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
+import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
+import {
+    CheckBitcoinSigs
+} from "@keep-network/bitcoin-spv-sol/contracts/CheckBitcoinSigs.sol";
+import "../Bridge.sol";
+
+library FraudVerification {
+    using BytesLib for bytes;
+    using BTCUtils for bytes;
+    using BTCUtils for uint32;
+
+    struct Parameters {
+        /// The amount of ETH the party challenging the wallet for fraud needs
+        /// to deposit.
+        uint256 fraudChallengeDepositAmount; //TODO: Initialize
```

Comment:
Maybe just `depositAmount`?

Reply:
The name comes from [rfc](https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-1.adoc#fraud-proofs).
We can change that, but we will probably have to change it in roc as well.

Reply:
Yeah, we do not need to have 1:1 naming, let's not worry about it. Eventually, we will have to write document with final parameter names for auditors but it's not a concern for this PR.

Reply:
Renamed parameters in b12e4a1e0c9e8b2bae96b0c374ed763688470abe.